### DISCUSSION TITLE
Export settings

### DISCUSSION THREAD

**Discussion by matar3** [RyotaUshio/obsidian-pdf-plus#204] [Q&A] [answered]
A plugin like pdf++ has quite a lot of settings, and when an issue arises it is always taking me a lot of time troubleshooting it (to understand which change in settings caused the issue).  Having a function of settings export would be of great help, because would let us get to a previous working state without having to uninstall and reinstall the plugin and reset all the settings from scratch. Could it be possible to export settings as a json file and then import settings through that file in obsidian?

**Comment by RyotaUshio** [comment, marked as answer]
All the settings are already saved as `.obsidian/plugins/pdf-plus/data.json`. So you can "export" the settings by just copying that json file, and you can "import" it by replacing the file and then reloading the plugin (don't forget the reload!).

**Reply by matar3** [reply]
Very good news to know. Thank you very much !
